raster-foundry/raster-foundry
# Rework scenes sort and index

## Overview

This PR adds a new index to the scenes table on `(coalesce(acquisition_date, created_at), id)` to
convince the database to use an index instead of a sequential scan for sorts. It also reworks
the default sort ordering so that the id sort is in the same direction as the first sort key from the
page request (because otherwise the db still won't use the index).

### Checklist


## Testing Instructions

Here's a sql query you're going to run a few times:

```sql
<...>
```

 * Run the query without creating the index against staging and note that the 
 * Manually create the index in the staging db using the migration sql
 * Run it again and note that the db decided to use the index this time
 * Run it again with the id sort the opposite direction from the `acquisition_date`/`created_at` coalesce sort and note that again the db chooses not to use the index

Closes #3188
